config/settings/base.py

```diff
@@ -34,14 +34,14 @@
 DEBUG = os.environ.get("DEBUG", "false") == "true"
 
 # Trust the X_FORWARDED_PROTO header from the proxy or load balancer so Django is aware it is accessed by https
-if "TRUST_FORWARDED_PROTO" in os.environ:
+if os.environ("TRUST_FORWARDED_PROTO", "no") == "yes":
```

Comment:
Do you ming using the same string values for the bool as the other variables in this file ? ("true", "false")